vmm/src/config.rs

```diff
@@ -2826,6 +2840,7 @@ mod tests {
             rate_limiter_config: None,
             pci_segment: 0,
             serial: None,
+            affinity: None,
```

Comment:
It may be more self-explanatory to rename this `queue_affinity`.

Reply:
sure I think that makes sense